src/app/(proper_react)/redesign/(authenticated)/user/welcome/View.tsx

```diff
@@ -41,6 +42,25 @@ export const View = ({
   const skipInitialStep = stepId === "enterInfo";
   const [currentStep, setCurrentStep] = useState<StepId>(stepId);
   const router = useRouter();
+  const record = useTelemetry();
+
+  useEffect(() => {
+    let pageName = "welcome";
+    if (currentStep === "enterInfo") {
+      pageName = "enter_scan_info";
+    } else if (currentStep === "findExposures") {
+      pageName = "scanning_for_exposures";
+    }
+
+    record("page", "view", {
+      utm_campaign: "broker_scan",
+      utm_content: pageName,
+      utm_term: skipInitialStep ? "legacy_user" : "new_user",
```

Comment:
This doesn't seem correct to me either - UTM terms say something about the way the user _got to_ the current page, not about the current page itself. Instead, I think that when we report `page.view` events, we'll want to report the `path` of the page that is being viewed - i.e. the current path name. We can then pass in how the user got here (`utm_source`) and e.g. how they were targeted (`utm_campaign` for `legacy_user` vs `new_user`) in the UTM parameters.

`utm_term` represents search terms entered by the user, which I don't think is ever applicable for links we send ourselves.

Reply:
See discussion [below](https://github.com/mozilla/blurts-server/pull/3823#discussion_r1432837294).